platforms/android/AndroidManifest.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version='1.0' encoding='utf-8'?>
-<manifest android:hardwareAccelerated="true" android:versionCode="50" android:versionName="0.0.50" android:windowSoftInputMode="adjustPan" package="com.ocdevel.habitrpg" xmlns:android="http://schemas.android.com/apk/res/android">
+<manifest android:hardwareAccelerated="true" android:versionCode="51" android:versionName="0.0.51" package="com.ocdevel.habitrpg" xmlns:android="http://schemas.android.com/apk/res/android">
     <supports-screens android:anyDensity="true" android:largeScreens="true" android:normalScreens="true" android:resizeable="true" android:smallScreens="true" android:xlargeScreens="true" />
     <uses-permission android:name="android.permission.INTERNET" />
     <application android:hardwareAccelerated="true" android:icon="@drawable/icon" android:label="@string/app_name">
-        <activity android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale" android:label="@string/app_name" android:launchMode="singleTop" android:name="HabitRPG">
-            <intent-filter>
+        <activity android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale" android:label="@string/activity_name" android:launchMode="singleTop" android:name="CordovaApp" android:theme="@android:style/Theme.Black.NoTitleBar" android:windowSoftInputMode="adjustResize">
```

Comment:
Pls revert following properties:
- ```android:label```
- ```android:name```

and remove ```android:theme``` , because with ```NoTitleBar``` the DateTime Pickers gonna look like Android 2.3 or lower :D